src/extension.ts

```diff
@@ -99,7 +99,7 @@ const checkCurrentFiles = (showDecorations: boolean) => {
 const activateCodeActionStuff = (context: vscode.ExtensionContext) => {
   context.subscriptions.push(
     vscode.languages.registerCodeActionsProvider(
-      { pattern: '**/package.json' },
+      [{ pattern: '**/package.json' }, { pattern: '**/pnpm-workspace.yaml' }],
```

Comment:
I think it's always named `pnpm-workspace.yaml`, right?

Reply:
That is correct, per [the pnpm doccs](https://pnpm.io/pnpm-workspace_yaml).